doozer/doozerlib/cli/images_health.py

```diff
@@ -85,9 +132,10 @@ def url_text(url, text):
         if latest_success_idx == -1:
             # No success record was found
             msg += f'Failing for at least the last {len(records)} attempts / {oldest_attempt_bi_dt}'
+            msg += f'. See more details: {art_dash_link}'
         else:
             msg += f'Last {url_text(latest_success_bi_task_url, "success")} was {latest_success_idx} attempts ago on {latest_success_bi_dt}'
-
+        msg += f'. See more details: {art_dash_link}'
```

Comment:
use url_text to add hyperlink to the text instead of directly show the link url

Reply:
Ah i understand slack markup style now, your implementation is correct.